package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "ftx-api",
-  "version": "1.1.8",
+  "name": "ftx-api-typed",
+  "version": "1.2.3",
```

Comment:
Why `1.2.3`?

Reply:
+1. Should be same version or 1.1.9 (assuming no breaking changes).